libraries/AP_IOMCU/fw_uploader.cpp

```diff
@@ -60,13 +60,21 @@
 
 extern const AP_HAL::HAL &hal;
 
+#ifndef HAL_IOMCU_BOOTLOADER_BAUDRATE
+#define HAL_IOMCU_BOOTLOADER_BAUDRATE 115200
+#endif
+
+#ifndef HAL_IOMCU_CHIBIOS_BOOTLOADER
+#define HAL_IOMCU_CHIBIOS_BOOTLOADER 0
+#endif
+
 /*
   upload a firmware to the IOMCU
  */
 bool AP_IOMCU::upload_fw(void)
 {
     // set baudrate for bootloader
-    uart.begin(115200, 256, 256);
+    uart.begin(HAL_IOMCU_BOOTLOADER_BAUDRATE, 256, 256);
```

Comment:
```suggestion
    uart.begin(AP_IOMCU_BOOTLOADER_BAUDRATE, 256, 256);
```

... similarly for other defines introduced.  `AP_IOMCU` isn't part of the HAL.

Reply:
fixed